### PR TITLE
[ci] resolve ansible-lint errors

### PR DESCRIPTION
**Pull request by dwang**
I've been seeing some odd errors on Ansible recently and some new `ansible-lint` warnings popped up since the last PR. Hopefully, this solves all of them and builds can pass again.

For example: [Travis CI](https://travis-ci.org/openwisp/ansible-openwisp2/jobs/496595884#L621)

### CODE REVIEW

File: tasks/apt.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 - name: Update APT package cache
```
Comment (atb00ker) [inline tasks/apt.yml]: @hispanico 
Someone recently informed me it's not a good practice to include `update_cache` in role, rather it's better to add it in playbook directly like:

```
  pre_tasks:
    - name: update apt cache
      apt:
        update_cache: yes
        cache_valid_time: 86400
```
Reason:
Updating the package cache is very time consuming (you can even get blocked if you do it too frequently). For that reason it doesn't belong in individual roles, it should be done only once at the start of the playbook.

Can you please comment on the same? Should we change accordingly?

Thanks

File: tasks/apt.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
   apt: update_cache=yes
   changed_when: false
+  retries: 5
```
Comment (atb00ker) [inline tasks/apt.yml]: @dwang sorry, i am not really aware of what was causing the failure and how does this help, can you please share your research? 

Thanks.

File: tasks/apt.yml (hunk 1 above)
Comment (dwang) [reply tasks/apt.yml]: I'm not familiar with this, but I have seen that updating the package cache is time consuming. The change sounds beneficial to me.

File: tasks/apt.yml (hunk 2 above)
Comment (dwang) [reply tasks/apt.yml]: @atb00ker There was not a failure directly related to the playbook, but with `ansible-lint`. It was complaining that there was not a retry in the case that remote tasks failed.
For example:
```
[405] Remote package tasks should have a retry
/home/travis/.ansible/roles/role_under_test/tasks/apt.yml:1
Task/Handler: Update APT package cache
```

With the PR, I fixed some more `ansible-lint` errors. I'm not sure why the lint errors weren't present before, but it was causing builds for master and other PRs to fail.

Comment (atb00ker) [reply tasks/apt.yml]: Alright thanks. :+1:

File: tasks/apt.yml (hunk 1 above)
Comment (nemesisdesign) [reply tasks/apt.yml]: I prefer if we leave this how it is now, I'd fix this role and then start focusing the devops technical effort the dockerization project

File: tasks/apt.yml (hunk 2 above)
Comment (nemesisdesign) [reply tasks/apt.yml]: good call :+1:

**Pull request merged**